phpstan.neon

```diff
@@ -15,6 +15,15 @@ parameters:
         -
             message: "/^Parameter #1 \\$function of function register_shutdown_function expects callable\\(\\): void, 'register_shutdown…' given\\.$/"
             path: src/Transport/HttpTransport.php
+        -
+            message: '/^Class Http\\Adapter\\Guzzle6\\Client not found\.$/'
+            path: src/ClientBuilder.php
+        -
+            message: '/^Call to static method createWithConfig\(\) on an unknown class Http\\Adapter\\Guzzle6\\Client\.$/'
+            path: src/ClientBuilder.php
+        -
+            message: '/^Access to constant PROXY on an unknown class GuzzleHttp\\RequestOptions\.$/'
+            path: src/ClientBuilder.php
```

Comment:
Wouldn't be better to require-dev those packages? We would just have to add another CI job where we remove those to check that our code works even without them...

Reply:
Since this part is not even tested, adding more steps to the CI won't allow us to know if this is gonna work or not so it's kinda useless